src/Fixer/LanguageConstruct/FunctionToConstantFixer.php

```diff
@@ -45,12 +45,12 @@ final class FunctionToConstantFixer extends AbstractFixer implements Configurabl
     use ConfigurableFixerTrait;
 
     /**
-     * @var null|array<string, list<Token>>
+     * @var null|array<string, non-empty-list<Token>>
      */
     private static ?array $availableFunctions = null;
 
     /**
-     * @var array<string, list<Token>>
+     * @var array<string, non-empty-list<Token>>
```

Comment:
```suggestion
     * @var null|array<string, non-empty-list<Token>>
```

Reply:
I think it is never `null`. It is uninitialized or array.

Reply:
You do not document it. PHPStan wants that uninitialized properties are initialized in constructor, but you can see here that it also complains that the property never is null: https://phpstan.org/r/b1cf492b-d361-41ef-9748-d92827e40ab2

> Property HelloWorld::$a (string|null) is never assigned null so it can be removed from the property type.

Reply:
> you do not document it

it's not the way project follows.
i recommend to follow so far status quo for majority of cases, or change the approach (but apply it fully)

Reply:
> it's not the way project follows.

There are 48 properties using this approach (uninitialized by default, not nullable, and not initialized in constructor).
E.g. 4 of them [are here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/117a5b30c11120d955a68b4815e78e8bb3b0f7b0/src/Fixer/Whitespace/NoExtraBlankLinesFixer.php#L75-L87).

Usually I try to avoid this. Unitialized properties should be initialized in constructor. 
But in our fixers the `configure` method is similar to a constructor, and many properties are initialized there. So I'm not sure if it is the right approach for us to require the initialization in constructor.
And making all these properties nullable would create many new "null value" phpstan errors, especially when using a higher level.